### ISSUE TITLE
[linux] [3.15] Accessibility bus warning

### ISSUE THREAD

**Issue by ddw-js-70** [supercollider/supercollider#7679]
## Environment

* SuperCollider version: 3.15 dev
* Operating system: xubuntu 26.04
* Other details (Qt version, audio driver, etc.):

## Steps to reproduce

During startup:

```supercollider
(sclang:731115): dbind-WARNING **: 10:20:28.269: Couldn't connect to accessibility bus: Failed to connect to socket /root/.cache/at-spi/bus_0.0: Permission denied
```

Never heard of this... what's up? If it's important, it should be documented. If it isn't important, perhaps the message should be suppressed.

**Comment by dyfer** [comment]
I don't think we emit this ourselves - maybe it comes from Qt?

**Comment by jamshark70** [comment]
For what it's worth, on my machine, this message has disappeared. I have no idea why.

So I think I'll close this issue, and reopen if it happens again.
